tools/src/commands/CloseLinearIssueFromGithub.ts

```diff
@@ -0,0 +1,66 @@
+import { Command } from '@expo/commander';
+
+import * as GitHub from '../GitHub';
+import * as Linear from '../Linear';
+import logger from '../Logger';
+
+type ActionOptions = {
+  issue: string;
+};
+
+export default (program: Command) => {
+  program
+    .command('close-linear-issue-from-github')
+    .alias('clifg')
+    .description('Close a Linear issue imported from GitHub.')
+    .option('-i, --issue <string>', 'Number of the original GitHub issue.')
+    .asyncAction(action);
+};
+
+async function action(options: ActionOptions) {
+  if (isNaN(Number(options.issue))) {
+    throw new Error('Flag `--issue` must be provided with a number value');
+  }
+  if (!process.env.GITHUB_TOKEN) {
+    throw new Error('Environment variable `GITHUB_TOKEN` is required for this command.');
+  }
+  if (!process.env.LINEAR_API_KEY) {
+    throw new Error('Environment variable `LINEAR_API_KEY` is required for this command.');
+  }
+
+  try {
+    await closeIssueAsync(+options.issue);
+  } catch (error) {
+    logger.error(error);
+    throw error;
+  }
+}
+
+async function closeIssueAsync(githubIssueNumber: number) {
+  const linearIssues = await Linear.getIssuesAsync({
+    teamId: Linear.ENG_TEAM_ID,
+    filter: {
+      description: {
+        containsIgnoreCase: `This issue was automatically imported from GitHub: [https://github.com/expo/expo/issues/${githubIssueNumber}]`,
```

Comment:
maybe we can just check for the issue link in the description and the Github label? then if we adjust the text at all it's a bit more resistant to breaking

Reply:
Sure, I've just pushed a commit updating this